src/index.html

```diff
@@ -3,11 +3,311 @@
+            alt="Speaker square"
+          >
+
+          <h3 class="categories__name">Speakers</h3>
+        </div>
+
+        <div class="categories__category">
+          <img
+            class="categories__photo categories__photo--5"
+            src="images/photos/categories/sunglasses/sunglasses-blue-320.jpg"
+            alt="Sunglasses blue"
+          >
+
+          <img
+            class="categories__photo categories__photo--6"
+            src="images/photos/categories/sunglasses/sunglasses-black-320.jpg"
+            alt="Sunglasses black"
+          >
+
+          <h3 class="categories__name">Audio sunglasses</h3>
+        </div>
+
+        <div class="button-wrapper">
+          <button class="button-wrapper__button">View all</button>
+        </div>
+      </section>
+
+      <section id="buy-reasons" class="buy-reasons">
+        <div class="buy-reasons__layout">
+          <h1 class="buy-reasons__title">Why buy direct from Bose</h1>
+
+          <div class="buy-reasons__content">
+            <h3 class="buy-reasons__why">Free 2-day shipping and returns</h3>
+
+            <h3 class="buy-reasons__why">90-day risk-free trial</h3>
+
+            <h3 class="buy-reasons__why">World class customer service</h3>
+
+            <h3 class="buy-reasons__why">My Bose account management</h3>
+
+            <p class="buy-reasons__praise">
+              A great product is more than what’s
+              in the box. It’s a promise of premium
+              performance, world-class support,
+              and everything you expect from a
+              trusted brand. It’s just one of many
+              reasons why you’ll shop with
+              confidence on Bose.com.
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section id="contact" class="contact">
+        <h1 class="contact__title">Contact us</h1>
+
+        <div class="contact__content">
+          <form
+            class="contact__form"
+            method="post"
+          >
+            <label class="contact__label" for="name">Name:</label>
+
+            <input
+              id="name"
+              class="contact__input"
+              type="text"
+              placeholder="Name"
+              required
+            >
+
+            <label class="contact__label" for="email">Email:</label>
+
+            <input
+              id="email"
+              class="contact__input"
+              type="email"
+              placeholder="Email"
+              required
+            >
+
+            <label class="contact__label" for="message">Message:</label>
+
+            <textarea
+              id="message"
+              class="contact__input contact__message"
+              type="text"
+              placeholder="Message"
+              required
+            ></textarea>
+
+            <div class="button-wrapper">
+              <button class="button-wrapper__button contact__send">Send</button>
+            </div>
```

Comment:
Add 'type="submit"' attribute to the 'button' tag to submit the form.

Reply:
I added type to button.